api/.env-sample

```diff
@@ -0,0 +1,6 @@
+NODE_ENV=development
+SERVER_PORT=3000
+DATABASE_HOST=
+DATABASE_USER=
+DATABASE_PASSWORD=
+DATABASE_NAME=
```

Comment:
total nit pick but adding a newline at the end of every file is nice - that way you can `cat` it cleanly.